extra/nrnivmodl-core.in

```diff
@@ -28,8 +31,9 @@ while getopts "n:v:o:i:l:p:hV" OPT; do
     h)
         echo "$APP_NAME [options, ...] [mods_path]"
         echo "Options:"
-        echo "  -n <binary>     NMODL code generation compiler path"
-        echo "  -o <output_dir> Set the output dir (default: <arch>-core)"
+        echo "  -n <name>       The model name, used as a suffix in the shared library"
+        echo "  -m <mod2c_bin>  NMODL code generation compiler path"
+        echo "  -I <install_dir> Install to dir. Default: off - Only build in {ARCH}"
```

Comment:
@ferdonline : nitpick : as we are using `-i` can we use different letter here? something like `-d` for destination or `-p` for path?